pkg/operator/ceph/cluster/mgr/spec_test.go

```diff
@@ -260,3 +260,28 @@ func TestApplyPrometheusAnnotations(t *testing.T) {
 	assert.Equal(t, 1, len(c.annotations))
 	assert.Equal(t, 0, len(d.ObjectMeta.Annotations))
 }
+
+func TestCephMgrOrchestratorModuleEnvs(t *testing.T) {
+	c := New(
```

Comment:
If you're only testing the number of env vars, you should be able to create a simpler `cluster` object directly without calling `New()` with all the args. Or maybe no need for this test since testing for the number of args hasn't been an effective unit test in other places in rook.

Reply:
Right and also, it's tested elsewhere already.